contrib/OpenInExplorer.lua

```diff
@@ -40,6 +40,8 @@ local dsys = require "lib/dtutils.system"
 local proper_install = false
 if dt.configuration.running_os == "windows" then
 	proper_install = true
+else
+	dt.print_error('OpenInExplorer plug-in only supports Windows OS at this time')
 end

```

Comment:
Just add return above this and the script won't install

Reply:
I opted to add it below the error message so the user would get some visible feedback that the script wasn't loaded